examples/multi_modal/generate.py

```diff
@@ -0,0 +1,181 @@
+import os
+import random
+from multiprocessing import Pool, cpu_count
+
+import names
+import pandas as pd
+from fpdf import FPDF
+from pdf2image import convert_from_path
+
+# Templates for each class
+templates = {
+    "Cancelation": """\
+    Cancelation Request
+    -------------------
+    PRENAME: {prename}
+    SURNAME: {surname}
+    DATE: {date}
+
+    I would like to cancel my subscription/service.
```

Comment:
Maybe we could have use ChatGPT to augment the text there.

Reply:
This is just a toy example, if you really want to use a model like this you have to train in on your company data or a fitting opensource dataset